### PR TITLE
Added XPLA

### PR DESCRIPTION
**Pull request by rodri-r**

### CODE REVIEW

File: xpla/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# XPLA
+
+| | |
+|---|---|
+|Version|`v1.4.1`|
+|Binary|`xplad`|
+|Directory|`.xpla`|
+|ENV namespace|`XPLAD`
```
Comment (tombeynon) [inline xpla/README.md:8]:
```suggestion
|ENV namespace|`XPLAD`|
```

**Review by tombeynon — COMMENTED** [review]
Looks great, nice work!

**Comment by tombeynon** [comment]
Pre-release image: `ghcr.io/akash-network/cosmos-omnibus:v0.4.13-rc1-xpla-v1.4.1`

**Comment by rodri-r** [comment]
> Looks great, nice work!

Awesome thank you!

**Comment by rodri-r** [comment]
> Pre-release image: `ghcr.io/akash-network/cosmos-omnibus:v0.4.13-rc1-xpla-v1.4.1`

Don't believe I have to do anything here, correct?

**Pull request merged**